CHANGES.md

```diff
@@ -1,3 +1,9 @@
+# 0.22.0
+
+-   Rewrite of decimal-converter, added `allowNegative` parameter, handle input
+    `"."` as errorous
```

Comment:
I suggest adding: "It now takes a single options argument that allows you to configure it".